code/modules/projectiles/guns/ego_gun/aleph.dm

```diff
@@ -26,17 +26,36 @@
 	It’s the byproduct of some horrid experiment in a certain laboratory that eventually failed."
 	icon_state = "adoration"
 	inhand_icon_state = "adoration"
-	special = "This weapon does black damage over time on direct hit. \
-				This weapon has an AOE attack dealing 30 damage."
+	special = "Use in hand to swap between AOE, DOT and shotgun modes."
 	ammo_type = /obj/item/ammo_casing/caseless/ego_adoration
 	weapon_weight = WEAPON_HEAVY
 	fire_sound = 'sound/effects/attackblob.ogg'
 	fire_sound_volume = 50
-	fire_delay = 2.4 SECONDS
+	fire_delay = 21
 
 	attribute_requirements = list(
 							FORTITUDE_ATTRIBUTE = 80,
 							PRUDENCE_ATTRIBUTE = 80,
 							TEMPERANCE_ATTRIBUTE = 100,
 							JUSTICE_ATTRIBUTE = 80
 							)
+	var/mode = 0
+
+/obj/item/gun/ego_gun/adoration/attack_self(mob/user)
+	..()
+	switch(mode)
+		if(0)
+			to_chat(user,"<span class='warning'>You focus, changing for a DOT blast</span>")
+			ammo_type = /obj/item/ammo_casing/caseless/ego_adoration/dot
```

Comment:
This, to my knowledge, does not change currently loaded projectile(casing) type.

Reply:
It does, actually. I tested it on local/